config/locales/forms/es.yml

```diff
@@ -134,7 +134,7 @@ es:
     two_factor:
       backup_code: Código de seguridad
       # removed "security code" form labels as they trigger ios credit card reader
-      code: Código de un solo uso
+      code: Código único
```

Comment:
This string is used on the IAL1 (authentication) MFA pages, which weren't included in the pull request screenshots.

https://github.com/18F/identity-idp/blob/34f0bccd457e3ee7fe103f7e073d5fa278e9127c/app/views/two_factor_authentication/otp_verification/show.html.erb#L14

https://github.com/18F/identity-idp/blob/34f0bccd457e3ee7fe103f7e073d5fa278e9127c/app/views/two_factor_authentication/totp_verification/show.html.erb#L9

I _think_ we want to be consistent here (related ticket [LG-7932](https://cm-jira.usa.gov/browse/LG-7932)), so maybe not a problem, but flagging in case it wasn't intentional.

Reply:
Good catch, I believe the intention is to update language throughout.

Reply:
[I mentioned this to the team in Slack](https://gsa-tts.slack.com/archives/C01710KMYUB/p1667495585951079) and there don't seem to be any concerns, so should be good! (And I agree with being consistent in the language)